resourseleak/TestCleanupAnnotationBuggy.java

```diff
@@ -0,0 +1,14 @@
+package resourceleak;
+
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+
+
+public class TestCleanupAnnotationBuggy {
+
+    public static void testCleanupAnnotationBuggy(String args[])throws IOException {
+        String indexFile = args[1];
+        PrintStream printStream = new PrintStream(new FileOutputStream(indexFile));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *printStream*. Currently, there are execution paths that do not contain closure statements. Either a) close *printStream* in a try-finally block or b) close the resource by declaring *printStream* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).